geomesa-hbase/geomesa-hbase-datastore/src/test/scala/org/locationtech/geomesa/hbase/data/HBaseDataStoreTest.scala

```diff
@@ -22,6 +27,9 @@ import org.junit.runner.RunWith
 import org.locationtech.geomesa.features.ScalaSimpleFeature
 import org.locationtech.geomesa.filter.function.ProxyIdFunction
 import org.locationtech.geomesa.hbase.data.HBaseDataStoreParams._
+import org.locationtech.geomesa.hbase.proto.GeoMesaProto.{GeoMesaCoprocessorRequest, GeoMesaCoprocessorResponse, GeoMesaCoprocessorService}
+import org.locationtech.geomesa.hbase.rpc.coprocessor.GeoMesaCoprocessor.RpcCallbackImpl
+import org.locationtech.geomesa.hbase.rpc.coprocessor.{GeoMesaHBaseCallBack, GeoMesaHBaseRpcController}
```

Comment:
could you clean up the imports here? i assume this file could just be reverted now